htdocs/core/tpl/admin_extrafields_edit.tpl.php

```diff
@@ -143,7 +143,7 @@ function init_typeoffields(type)
 <input type="hidden" name="attrname" value="<?php echo $attrname; ?>">
 <input type="hidden" name="action" value="update">
 <input type="hidden" name="rowid" value="<?php echo (empty($rowid) ? '' : $rowid) ?>">
-<input type="hidden" name="enabled" value="<?php echo $extrafields->attributes[$elementtype]['enabled'][$attrname]; ?>">
+<input type="hidden" name="enabled" value="<?php echo htmlspecialchars($extrafields->attributes[$elementtype]['enabled'][$attrname]); ?>">
```

Comment:
We should not use htmlspecialchars
Rest of dolibarr code is using 
`dol_escape_htmltag($extrafields->attributes[$elementtype]['enabled'][$attrname]) `
Does it work for you with this ?